rom/dev/README.md

```diff
@@ -631,19 +667,18 @@ The following are the pre-conditions that should be satisfied:
 - Header is the only signed component. There are two signatures generated for the header.
 - First signature is generated using one of the manufacturing keys.
 - Second signature is generated using the owner public key.
-- To validate the header, hash and sign it using the selected manufacturer key and compare with the manufacturer signature in the preamble.
-- If the manufacturer signature matches, proceed with the owner signature validation. If the signature does not match, fail the validation.
-- The hash is already generated. Sign the above hash using the owner public key and match the signature. If the signature matches, the validation succeeds. If the signature does not match the validation fails.
+- To validate the header, hash and sign it using the selected ECC manufacturer key and compare with the ECC manufacturer signature in the preamble. 
```

Comment:
We cannot "hash and sign" the header since we do not have the private key, but we can verify that the signature was for the hash of the header and matches the specified public key.